ports/veigar/vcpkg.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "veigar",
+  "version": "1.2",
+  "maintainers": "winsoft666 <winsoft666@outlook.com>",
+  "description": "Cross platform RPC library using shared memory.",
+  "homepage": "https://github.com/winsoft666/veigar",
+  "license": "GPL-3.0-or-later",
+  "supports": "(windows | linux) & !uwp",
```

```suggestion
  "supports": "linux | (windows & !uwp & !xbox)",
```
if it really doesn't support osx or android.